File: Java/MonolithicApp/src/main/java/com/example/route/dto/RouteResponse.java
package com.example.route.dto;

import java.util.List;

public class RouteResponse {
    private String from;
    private String to;
    private int stops;
    private java.util.List<String> path;

    public RouteResponse(String from, String to, int stops, java.util.List<String> path) {
        this.from = from; this.to = to; this.stops = stops; this.path = path;
    }
    public String getFrom() { return from; }
    public String getTo() { return to; }
    public int getStops() { return stops; }
    public java.util.List<String> getPath() { return path; }
}

File: NET/MicroserviceApp/RouteService/DTOs/RouteResponse.cs
using System.Collections.Generic;

namespace RouteService.DTOs
{
    public class RouteResponse
    {
        public string From { get; set; }
        public string To { get; set; }
        public int Stops { get; set; }
        public List<string> Path { get; set; }

        public RouteResponse(string from, string to, int stops, List<string> path)
        {
            From = from;
            To = to;
            Stops = stops;
            Path = path;
        }
    }
}
